src/main/kotlin/no/nav/familie/ef/sak/behandling/RevurderingService.kt

```diff
@@ -84,6 +84,12 @@ class RevurderingService(
         ) {
             "Kan ikke opprette revurdering med årsak g-omregning for ${fagsak.stønadstype}"
         }
+        feilHvis(
+            fagsak.stønadstype != StønadType.BARNETILSYN &&
+                revurderingInnhold.behandlingsårsak == BehandlingÅrsak.SATSENDRING
```

Comment:
Burde vi ha en og samme årsak for både OT og BT? 
Samtidig er det kanskje fint å separere de?

Reply:
Mye likt, men samtidig veldig forskjellig. Jeg stemmer for å skille mellom g-omregning og satsendring som årsak til endring. (Hvis det var det du mente?)